internal/ewma/ewma_test.go

```diff
@@ -0,0 +1,41 @@
+package ewma
+
+// Copyright (c) 2013 VividCortex, Inc. All rights reserved.
+// Please see the LICENSE file for applicable license terms.
+
```

Comment:
We need to make sure to preserve the entire project's proper copyright and not make any edits. AFAIK @vadmeste they provide an interface is it not possible to implement what we want?

Reply:
@harshavardhana This will be removed.